app/views/questionnaire.py

```diff
@@ -281,6 +283,20 @@ def _save_sign_out(collection_id, eq_id, form_type, q_manager, this_location):
         return redirect(url_for('.get_sign_out', eq_id=eq_id, form_type=form_type, collection_id=collection_id))
 
 
+def _compare_and_remove_repeating_on_household_if_answers_changed(answer_store, group_id):
```

Comment:
It would be nicer if this just compared and returned whether it was true or not otherwise I think it's doing too much. Something like

```
if not _same_answers(…)
    _remove_repeating_on_household_answers(answer_store, group_id)
```